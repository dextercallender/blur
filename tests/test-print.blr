/* Print test */

print("hello");
print(12345);
print(1.12345);
print('a');
