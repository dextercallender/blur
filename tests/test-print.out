hello123451.12345a
